reinforcement_learning/bandits_recsys_movielens_testbed/bandits_movielens_testbed.ipynb

```diff
@@ -54,6 +54,16 @@
     "To get started, we'll import the Python libraries we need, set up the environment with a few prerequisites for permissions and configurations."
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Install sagemaker SDK v2\n",
+    "!pip install sagemaker==2.16.1"
```

Comment:
Need to check if we need this anymore. I thought the Python SDK v2 was the new default for the notebooks.